enterprise-suite/scripts/lbc_test.py

```diff
@@ -206,6 +206,20 @@ def test_helm_args(self):
         expect_cmd(r'helm install es-repo/enterprise-suite --name enterprise-suite --namespace lightbend --devel --values \S+ --set minikube=true --fakearg')
         lbc.main(['install', '--skip-checks', '--creds='+self.creds_file, '--delete-pvcs', '--', '--set', 'minikube=true', '--fakearg'])
 
+    def test_helm_args_namespace(self):
+        expect_cmd(r'helm repo add es-repo https://repo.lightbend.com/helm-charts')
+        expect_cmd(r'helm repo update')
+        expect_cmd(r'helm status enterprise-suite', returncode=-1)
+        expect_cmd(r'helm install es-repo/enterprise-suite --name enterprise-suite --namespace foobar --devel --values \S+ --set minikube=true --fakearg')
+        lbc.main(['install', '--skip-checks', '--creds='+self.creds_file, '--delete-pvcs', '--', '--set', 'minikube=true', '--fakearg', '--namespace=foobar'])
+
+    def test_helm_args_namespace_val(self):
```

Comment:
I'd also like to see a test with no `--namespace` value specified in the `lbc.py` args.  And maybe one where the `lbc.py` one and the helm one differ.

Reply:
+1